packages/itwinui-variables/index.scss

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import './mixins';
+
+:root {
```

Comment:
Maybe you know the pros of using `:root` here?

Reply:
`:root` is interchangeable with `html` usually. Most places I've seen use `:root` to declare CSS variables.

Two differences I'm aware of:
- `:root` has higher specificity than `html`. I think that's fine in this case.
- `:root` refers to the root of the shadow DOM if used there, but it doesn't apply in this case.

Reply:
Yeah, I was aware of the 2nd point..
First one didnt know.. interesting.

Reply:
In recent tweets I saw someone changing from `:where(html)` to `:host` in order to support web components